### PR TITLE
Terse notes for board meeting 2022-10-24

### PR DESCRIPTION
**Pull request by user-group**

### CODE REVIEW

File: content/about/board/2022-10-24.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (spier) [inline content/about/board/2022-10-24.md:8]:
```suggestion
---
layout: page
show_meta: false
title: '2022-10-24'
---

```

Looks like you need this meta info at the beginning of the file to make it render correctly on the website.

File: content/about/board/2022-10-24.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Date and Time of Meeting
```
Comment (spier) [inline content/about/board/2022-10-24.md]:
```suggestion
### Date and Time of Meeting

```

Changing headline to what I see in previous reports.
Also adding spacing.

File: content/about/board/2022-10-24.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# Date and Time of Meeting 
+2022-10-24, 7 p.m. - 8 p.m. CEST / 12 p.m. - 1 p.m. CDT
+
+* Call to order 19:11 CEST
+* Adjourned 19:39 CEST
+* Next meeting 2022-11-28, 7 p.m-8 p.m. CEST / 12 p.m. - 1 p.m. CDT
+
+# Roll call
+
+## Directors and Officers Present
```
Comment (spier) [inline content/about/board/2022-10-24.md]:
```suggestion
### Roll call

#### Directors and Officers Present

```

Changing headline to what I see in previous reports.
Also adding spacing.

File: content/about/board/2022-10-24.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Maximiliano Capraro (Assistant Treasurer)
+Isabel Drost-Fromm (President)
+Daniel Izquierdo Cortazar (Vice President)
+Russell Rutledge (Assistant Secretary)
+Thomas Sadler (Chair)
+Sebastian Spier
+Klaas-Jan Stol
+Johannes Tigges
+
+## Directors and Officers Absent
```
Comment (spier) [inline content/about/board/2022-10-24.md]:
```suggestion
#### Directors and Officers Absent

```

Changing headline to what I see in previous reports.
Also adding spacing.

File: content/about/board/2022-10-24.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+Danese Cooper (President)
+Jakob Green
+Georg Grütter
+
+
+## Members Present
```
Comment (spier) [inline content/about/board/2022-10-24.md]:
```suggestion
#### Members Present

```

Changing headline to what I see in previous reports.
Also adding spacing.

File: content/about/board/2022-10-24.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+
+# Date and Time of Meeting 
+2022-10-24, 7 p.m. - 8 p.m. CEST / 12 p.m. - 1 p.m. CDT
+
+* Call to order 19:11 CEST
+* Adjourned 19:39 CEST
+* Next meeting 2022-11-28, 7 p.m-8 p.m. CEST / 12 p.m. - 1 p.m. CDT
```
Comment (tsadler1988) [inline content/about/board/2022-10-24.md]:
```suggestion
* Next meeting 2022-11-28, 7 p.m-8 p.m. CET / 12 p.m. - 1 p.m. CST
```
Daylight savings will be over by this date.

File: content/about/board/2022-10-24.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+## Directors and Officers Absent
+Danese Cooper (President)
+Jakob Green
+Georg Grütter
+
+
+## Members Present
+Clare Dillon (Executive Director)
+
+# Votes Taken
```
Comment (spier) [inline content/about/board/2022-10-24.md]:
```suggestion
### Votes Taken

```

Changing headline to what I see in previous reports.
Also adding spacing.

**Review by spier — CHANGES_REQUESTED** [review]
I adapted the formatting based on what I found in a [previous terse notes](https://raw.githubusercontent.com/InnerSourceCommons/innersourcecommons.org/master/content/about/board/2022-09-22.md).

Please review.

**Review by tsadler1988 — CHANGES_REQUESTED** [review]
Thanks @user-group! Just one commit suggestion re: timezones for next meeting.

**Review by user-group — COMMENTED** [review]
i can't seem to click 'approve', so I'm just leaving this comment here as not sure what to do.

**Review by user-group — COMMENTED** [review]
can only comment, can't seem to approve. not sure how to proceed (please help :-))

**Comment by rrrutledge** [comment]
> can't seem to approve

Someone else has to approve.  You’re not allowed to approve your own PR :)

**Review by lenucksi — COMMENTED** [review]
Looks fine, not sure with some of the role assignments though (i.e. chair and president). Can someone take a second look in the vote register please?)

**Comment by user-group** [comment]
@lenucksi I assumed 'chair' was the meeting chair, but maybe that's not what I should have done? I think I copied all others from previous notes.

File: content/about/board/2022-10-24.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+* Call to order 19:11 CEST
+* Adjourned 19:39 CEST
+* Next meeting 2022-11-28, 7 p.m-8 p.m. CET / 12 p.m. - 1 p.m. CST
+
+### Roll call
+
+#### Directors and Officers Present
+
+Silona Bonewald (Treasurer)
+Maximiliano Capraro (Assistant Treasurer)
```
Comment (maxcapraro) [inline content/about/board/2022-10-24.md]:
```suggestion
Maximilian Capraro (Assistant Treasurer)
```

I do like the sound of it, though :-)

**Review by maxcapraro — COMMENTED** [review]
Thanks for writing such excellent terse notes, @user-group :-) I proposed one tiny change inline, other than this it's good to go from my side 🚀

File: content/about/board/2022-10-24.md (hunk 8 above)
Comment (rrrutledge) [reply content/about/board/2022-10-24.md]: It does have a nice ring to it!

File: content/about/board/2022-10-24.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+### Roll call
+
+#### Directors and Officers Present
+
+Silona Bonewald (Treasurer)
+Maximilian Capraro (Assistant Treasurer)
+Isabel Drost-Fromm (President)
+Daniel Izquierdo Cortazar (Vice President)
+Russell Rutledge (Assistant Secretary)
+Thomas Sadler (Chair)
```
Comment (tsadler1988) [inline content/about/board/2022-10-24.md]:
```suggestion
Thomas Sadler (Secretary)
```

File: content/about/board/2022-10-24.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+Sebastian Spier
+Klaas-Jan Stol
+Johannes Tigges
+
+#### Directors and Officers Absent
+
+Danese Cooper (President)
```
Comment (tsadler1988) [inline content/about/board/2022-10-24.md]:
```suggestion
Danese Cooper (Chair)
```

**Review by tsadler1988 — CHANGES_REQUESTED** [review]

**Comment by tsadler1988** [comment]
> @lenucksi I assumed 'chair' was the meeting chair, but maybe that's not what I should have done? I think I copied all others from previous notes.

Yeah I think it's the individual's officer role rather than what they did in the meeting. I've made commit suggestions to fix this  :)

**Review by tsadler1988 — APPROVED** [review]

**Comment by tsadler1988** [comment]
@spier I think your suggestions have been incorporated, if you could approve this please?

**Review by spier — APPROVED** [review]

**Review by lenucksi — APPROVED** [review]
Looks fine to me. We still need 2-3 additional board members to sign off here. @MaineC @dicortazar @rrrutledge

**Comment by tsadler1988** [comment]
> Looks fine to me. We still need 2-3 additional board members to sign off here. @MaineC @dicortazar @rrrutledge

Previous terse notes haven't had more than 2 or 3 reviews so I think this is fine to merge.

**Comment by rrrutledge** [comment]
> I think this is fine to merge

I agree!  These aren’t terribly complicated or controversial, so just getting them in makes sense.

**Comment by lenucksi** [comment]
> > I think this is fine to merge
> 
> I agree! These aren’t terribly complicated or controversial, so just getting them in makes sense.

Good. Should we then add that to our meeting checklist?

**Comment by rrrutledge** [comment]
In the [meeting checklist](https://github.com/InnerSourceCommons/foundation/blob/master/.github/ISSUE_TEMPLATE/board-meeting.md) it just says to merge in the week following the meeting.  Should we adjust somehow?

**Comment by lenucksi** [comment]
> In the [meeting checklist](https://github.com/InnerSourceCommons/foundation/blob/master/.github/ISSUE_TEMPLATE/board-meeting.md) it just says to merge in the week following the meeting. Should we adjust somehow?

I think so. This differs from the practical/reality approach and thus we should update it with this. Make a PR invite the people, conclude & hopefully merge and have up-to-date material for the future.

**Comment by rrrutledge** [comment]
I think that this PR was an exception since it was the first time that Klaas is working on it.  If you look at the months beforehand, these notes get an approval within a day or two of being opened.  The longest delays are getting the PR opened in the first place and then hitting the merge button after approval.

**Comment by lenucksi** [comment]
> I think that this PR was an exception since it was the first time that Klaas is working on it. If you look at the months beforehand, these notes get an approval within a day or two of being opened. The longest delays are getting the PR opened in the first place and then hitting the merge button after approval.

I was more suggesting to add the 2 or 3 reviews suffice as @tsadler1988 pointed out here; https://github.com/InnerSourceCommons/innersourcecommons.org/pull/390#issuecomment-1314709510
What do you think about that?

**Comment by rrrutledge** [comment]
Ah, makes sense!

**Pull request merged**